proposals/0304-structured-concurrency.md

```diff
@@ -535,41 +548,37 @@ It is possible to obtain a task that the handle refers to by using `handle.task`
 
 ```swift
 extension Task.Handle { 
-  /// Returns the task object the handle is refering to.
+  /// Returns the task object the handle is referring to.
   var task: Task { get }
 }
 ```
 
-Getting the handle's task allows us to check if the work we're about to wait on perhaps was already cancelled (by calling `handle.task.isCancelled`), or query at what priority the task is executing.
+Getting the handle's task allows us to check if the work we're about to wait on perhaps was already cancelled (by calling `handle.isCancelled`), or query at what priority the task is executing.
 
 #### Detached tasks
 
-A new, detached task can be created with the `Task.runDetached` operation. The resulting task is represented by a `Task.Handle`.
+A new, detached task can be created with the `spawnDetached` operation. The resulting task is represented by a `Task.Handle`.
 
 ```swift
-extension Task {
-  /// Create a new, detached task that produces a value of type `T`.
-  @discardableResult
-  static func runDetached<T: ConcurrentValue>(
-    priority: Priority = .default,
-    startingOn executor: UnownedExecutorRef? = nil,
-    operation: @escaping @concurrent () async -> T
-  ) -> Handle<T>
-
-  /// Create a new, detached task that produces a value of type `T` or throws an error.
-  @discardableResult
-  static func runDetached<T: ConcurrentValue>(
-    priority: Priority = .default,
-    startingOn executor: UnownedExecutorRef? = nil,
-    operation: @escaping @concurrent () async throws -> T
-  ) -> Handle<T, Error>
-}
+/// Create a new, detached task that produces a value of type `T`.
+@discardableResult
+static func runDetached<T: Sendable>(
```

Comment:
Looks like these still need to be changed to `spawnDetached`.

Reply:
Whoops, okey going through again now